packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -68,6 +68,34 @@ const defaultProps: Pick<Props, 'shouldMatchSomePermissions'> = {
 };
 
 const Authorized = (props: Props) => {
+  // Check first for data fences
```

Comment:
@emmenko  why do we need to first check for dataFences ? I would assume that General permissions have precedence.

Reply:
From the discussions I read yesterday from you I assumed that data fences take precedence and the other permissions are ignored.

If I understood it wrong, please feel free to adjust.

Reply:
in the permissions UI whenever there is a general permission for example to manage orders, any datafence is overruled by it, ideally `authorized` should mirror the same behaviour.

but in practice, since they are going to be mutually exclusive (https://github.com/commercetools/merchant-center-services/pull/686) the order does not matter since if there is no datafence then the check for `actualDataFences` would fail anyway and that block of code would not run.

Reply:
True however wouldn't it be confusing if you pass both and only see one being evaluated? Should we maybe also log/reportToSentry in that case saying: hej, only one of those two work?

Reply:
There are to people in this scenario, there is the developer of the app that demands both general permissions and dataFences (which is legit as he does not know what kind of permission the end user has). and then there is the end user with a specific kind of permission (either general one or dataFence, but not both).

We can think of `demanded*` as an OR combination, then it should not be confusing to pass both and only see one being evaluated.